lib/mixlib/shellout/windows/core_ext.rb

```diff
@@ -303,30 +311,83 @@ def create(args)
               startinfo,              # Startup Info
               procinfo                # Process Info
             )
+
+            unless bool
+              if FFI.errno == ERROR_PRIVILEGE_NOT_HELD
+                raise SystemCallError.new("CreateProcessAsUserW (You must hold the 'Replace a process level token' permission. Refresh the logon context after adding this right to make it effective.)", FFI.errno)
```

Comment:
I'd replace "Refresh the logon context" with "Please sign out". Also for accuracy "Adjust Memory Quotas for a process" is also needed though admins will already have this with default settings.